server.js

```diff
@@ -47,6 +48,21 @@ if (process.env.FORCE_HTTPS === "true") {
 	app.use(enforce.HTTPS({trustProtoHeader: true}));
 }
 
+if (process.env.IP_WHITELIST) {
+        let clientIp = function(req, res) {
+                return req.headers['x-forwarded-for'] ? (req.headers['x-forwarded-for']).split(',').pop() : ""
+        }
+        let whitelist_ips = ['::1', '127.0.0.1'].concat(process.env.IP_WHITELIST.split(','))
+        app.use(
+                ipfilter(whitelist_ips, {
+                        detectIp: clientIp,
+                        forbidden: 'You are not authorized to access this page.',
```

Comment:
The error handling is the only tricky thing with this, and express in general.  Right now this message is not displayed, but instead an internal error message is shown alone on a blank page.  May need special handling for the IpDeniedError in our normal error handler.

Reply:
Also, I the Opts object you are passing here does not match the interface given by express-ipfilter:
`export interface IpFilterOptions {
  detectIp?: (request: express.Request) => Ip;
  excluding?: Route[];
  log?: boolean;
  logLevel?: 'all' | 'deny' | 'allow';
  mode?: 'deny' | 'allow';
  // `@types/proxy-addr` does not export the `trust` parameter type
  trustProxy?: any;
}`